### PR TITLE
Update link text

### PR DESCRIPTION
**Pull request by imba-tjd**
MSDN --> Default Marshaling Behavior

### CODE REVIEW

File: docs/standard/native-interop.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ Both of the above examples depend on parameters, and in both cases, the paramete
...
+The reason marshalling is needed is because the types in the managed and unmanaged code are different. In managed code, for instance, you have a `String`, while in the unmanaged world strings can be Unicode ("wide"), non-Unicode, null-terminated, ASCII, etc. By default, the P/Invoke subsystem will try to do the Right Thing based on the default behavior which you can see on [Default Marshaling Behavior](../../docs/framework/interop/default-marshaling-behavior.md). However, for those situations where you need extra control, you can employ the `MarshalAs` attribute to specify what is the expected type on the unmanaged side. For instance, if we want the string to be sent as a null-terminated ANSI string, we could do it like this:
```
Comment (rpetrusha) [inline docs/standard/native-interop.md]: This is an improvement, but the sentence doesn't read very well. Some suggestions:
- Right Thing --> right thing
- based on the default behavior which you...plus link --> based on the \[default behavior](../../docs/framework/interop/default-marshaling-behavior.md).
- MarshalAs can be a link: \[MarshalAs](xref:System.Runtime.InteropServicxes.MarshalAs)

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for improving this paragraph, @imba-tjd. I think that it can be improved still more, so I left some suggestions.

**Comment by imba-tjd** [comment]
OK, please review~

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @imba-tjd, for making the additional changes. I think that your changes have substantially improved this topic. I'll merge your PR now.

**Pull request merged**